courses/fundamentals_of_ada/adv_065_primitives.rst

```diff
@@ -0,0 +1,68 @@
+*********************
+Advanced Primitives
+*********************
+
+=================
+Type Derivation
+=================
+
+------------------------------
```

Comment:
This contradicts slide "General Rule For a Primitive"

Reply:
Do you mean it contradicts the rule of thumbs? Otherwise it is entirely possible to follow the given rules and end up with a primitive of several types.

```
package P is
  type T1 is new Integer;
  type T2 is new Integer;

  procedure S (A : T1, B : T2);
end;
```

I removed a typo sentence from the slide. LMK what you think

Reply:
No longer reads poorly, so I guess we're good :)